composer/loggers/mlflow_logger.py

```diff
@@ -184,6 +185,7 @@ def init(self, state: State, logger: Logger) -> None:
                 tags=self.tags,
                 log_system_metrics=self.log_system_metrics,
             )
+            self.run_url = f'{self.tracking_uri}/#/experiments/{self._experiment_id}/runs/{self._run_id}'
```

Comment:
Can you check if mlflow has an API for this? Also please include a manual test run so I can look at the link and make sure it works.

Reply:
No API replacements to be found :(

mcli logs test-mlflow-logger-pK4ZNT
<img width="1728" alt="image" src="https://github.com/mosaicml/composer/assets/76963540/aba005c7-9373-4983-a806-19a44f2c2daf">